src/test/java/org/hisp/dhis/rules/functions/RuleFunctionRightTests.java

```diff
@@ -29,101 +29,96 @@
  */
 
 import org.hamcrest.MatcherAssert;
-import org.hisp.dhis.rules.RuleVariableValue;
-import org.junit.Rule;
+import org.hisp.dhis.antlr.ParserExceptionWithoutContext;
+import org.hisp.dhis.rules.parser.expression.CommonExpressionVisitor;
+import org.junit.Before;
+import org.hisp.dhis.parser.expression.antlr.ExpressionParser;
 import org.junit.Test;
-import org.junit.rules.ExpectedException;
 import org.junit.runner.RunWith;
-import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.Map;
-
-import static java.util.Arrays.asList;
-import static org.hamcrest.CoreMatchers.is;
+ import org.hamcrest.CoreMatchers;
+import static org.mockito.Mockito.when;
 
 /**
  * @Author Zubair Asghar.
  */
 
-@RunWith( JUnit4.class )
+@RunWith( MockitoJUnitRunner.class )
 public class RuleFunctionRightTests
 {
-        @Rule
-        public ExpectedException thrown = ExpectedException.none();
+        @Mock
+        private ExpressionParser.ExprContext context;
 
-        private Map<String, RuleVariableValue> variableValues = new HashMap<>();
+        @Mock
+        private CommonExpressionVisitor visitor;
 
-        @Test
-        public void return_empty_string_for_null_input()
-        {
-                RuleFunction rightFunction = RuleFunctionRight.create();
-
-                MatcherAssert.assertThat( rightFunction.evaluate( Arrays.asList( null, "0" ), variableValues, null ),
-                    is( "" ) );
-                MatcherAssert.assertThat( rightFunction.evaluate( Arrays.asList( null, "10" ), variableValues, null ),
-                    is( "" ) );
-                MatcherAssert.assertThat( rightFunction.evaluate( Arrays.asList( null, "-10" ), variableValues, null ),
-                    is( "" ) );
+        @Mock
+        private ExpressionParser.ExprContext mockedFirstExpr;
+
+        @Mock
+        private ExpressionParser.ExprContext mockedSecondExpr;
+
+        @Before
+        public void setUp() {
+                when(context.expr(0)).thenReturn( mockedFirstExpr );
```

Comment:
minor style(spaces)

Reply:
If it is ok with you I will create a commit right after we finish the review to apply the codestyle to the whole project